docs/integration/kubernetes.md

```diff
@@ -823,3 +823,22 @@ Also, please make sure that your agent is [up to date](https://sematext.com/docs
 
 ### Why we need hostNetwork access and how to turn it off if desired
 Check out [our page about hostnetwork](https://sematext.com/docs/agents/sematext-agent/kubernetes/hostnetwork/).
+
+### Setting TLS Certificate Paths
```

Comment:
@yago-123 for some reason I missed that section. We are good with this.

Reply:
OK, asking for review for the final boss then